website/src/pages/api/messages/[id]/parent.ts

```diff
@@ -0,0 +1,48 @@
+import { getToken } from "next-auth/jwt";
+
+const handler = async (req, res) => {
+  const token = await getToken({ req });
+
+  // Return nothing if the user isn't registered.
+  if (!token) {
+    res.status(401).end();
+    return;
+  }
+
+  const { id } = req.query;
+
+  if (!id) {
+    res.status(400).end();
+    return;
+  }
+
+  const messageRes = await fetch(`${process.env.FASTAPI_URL}/api/v1/messages/${id}`, {
```

Comment:
Should this include `/parent` in the URL?

Reply:
I believe the backend does not provide the parent function..

Reply:
I read this a bit too quickly before going to take a bath.  You are right.  This seems to be the only way right now